alacritty/src/main.rs

```diff
@@ -28,6 +28,9 @@ use alacritty_terminal::sync::FairMutex;
 use alacritty_terminal::term::Term;
 use alacritty_terminal::tty;
 
+#[cfg(target_os = "macos")]
+use crate::macos::locale;
```

Comment:
Imports from modules inside of Alacritty are usually *below* those module imports.

You can see that in this file already.

Reply:
I'll move it, I just generally like to keep all my imports together.